cookiecutter/{{cookiecutter.organization}}-richie-site-factory/template/{{cookiecutter.site}}/src/frontend/package.json

```diff
@@ -23,10 +23,10 @@
     "richie-education": "2.14.1"
   },
   "devDependencies": {
-    "@formatjs/cli": "4.8.2",
-    "eslint": "8.9.0",
-    "nodemon": "2.0.15",
-    "prettier": "2.5.1",
-    "sass": "1.49.8"
+    "@formatjs/cli": "4.8.4",
+    "eslint": "8.14.0",
+    "nodemon": "2.0.16",
+    "prettier": "2.6.2",
+    "sass": "1.51.0"
```

Comment:
If cookiecutter directory is ignored, should those updates still be here ?

Reply:
You're right, nope as the `package.json` should be synced with the `yarn.lock`.